File: submissions/sm_033_surya/week_03/day_3/session_2/average_divisible_while.js
var sum = 0
var j = 0
var average
function average(limit,number) {
    
var i=1 
while (i<= limit)
{
    if (i % number == 0)
    {
        sum = sum+i
        j++
    }
    i++
}
average = sum / j 
    console.log(average)
}
average(100,5)

File: submissions/sm_009_chinmay/week_13/day_4/part_1/average_divisible_while.py
# Print the average of all the numbers from 0 to the given limit that are divisible by the given number (HINT: User while)

limit = int(input('Enter limit: '))
num = int(input('Enter number: '))
sum = 0
cnt = 0

while limit >= 0:
    if(limit % num == 0):
        sum = sum + limit
        cnt = cnt + 1
    limit = limit - 1
print(sum/cnt)